doc/examples/cloud-config-user-groups.txt

```diff
@@ -59,6 +59,8 @@ users:
 #           (the above command would create from stdin an SHA-512 password hash
 #           with 4096 salt rounds)
 #
+#          This field has no effect on existing users.
+#
```

Comment:
Similarly, this statement is not just applicable to the `passwd` field. Perhaps add some words after "Users are added after groups are added." further up in the file?

Reply:
Done, thanks.